src/App.js

```diff
@@ -38,6 +40,8 @@ function App() {
         <Route path="/seller-portal" element={<SellerPortal />} />
         <Route path="/buyer-portal" element={<BuyerPortal />} />
         <Route path="/properties" element={<PropertyList />} />
+        <Route path="/homepage" element={<HomePage />}/>
```

Comment:
could you use `/` instead of `/homepage`? it makes more sense this way.
otherwise the root of our website it empty.